exec/java-exec/src/main/java/org/apache/drill/exec/planner/physical/PlannerSettings.java

```diff
@@ -112,6 +112,12 @@
   public static final EnumeratedStringValidator QUOTING_IDENTIFIERS = new EnumeratedStringValidator(
       QUOTING_IDENTIFIERS_KEY, Quoting.BACK_TICK.string, Quoting.DOUBLE_QUOTE.string, Quoting.BRACKET.string);
 
+  /*
+    "planner.enable_unnest_lateral" is to allow users to choose enable unnest+lateraljoin feature.
+   */
+  public static final String ENABLE_UNNEST_LATERAL_KEY = "planner.enable_unnest_lateral";
```

Comment:
Do we need this option ? Or is this temporary ?

Reply:
This is temporary or for test purpose